onnxscript/rewriter/llama_rule_sets.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
+
+import onnxscript.ir as ir
+import onnxscript.rewriter.no_op as no_op
+import onnxscript.rewriter.pattern as orp
+from onnxscript.rewriter import pattern
+
+_op = pattern.onnxop
+
+
+def transpose_identity(x, perm):
+    return _op.Transpose(x, perm=perm)
+
+
+def transpose_identity_check(x: ir.Value, perm: ir.Attr | ir.RefAttr) -> bool:
```

Comment:
```suggestion
def _transpose_identity_check(x: ir.Value, perm: ir.Attr | ir.RefAttr) -> bool:
```

Can this be private?

Reply:
I prefer to keep the same prefix for the three functions. There will be 20 functions in this file. It seems better to keep the same prefix for the three functions related to the same rule. I wanted to group them in a class but I was sure onnxscript would allow that.